README.md

```diff
@@ -87,6 +87,9 @@ Add the contract address to the `.env` file in your root directory.
 |-----------|-------------|
 | `CONTRACT_ADDRESS` | Address of the light client contract deployed |
 
+
+**Note:** if you see the error `Error: Failed to send transaction`, check that your account has sufficient funds or check that the private key is valid.
```

Comment:
**Note**: If you see the error, check that your account has sufficient funds or check that the private key is valid.